Acceleratio.XCellKit/ExcelExport/Chart/Chart.cs

```diff
@@ -0,0 +1,192 @@
+﻿using DocumentFormat.OpenXml;
+using DocumentFormat.OpenXml.Drawing.Charts;
+using DocumentFormat.OpenXml.Packaging;
+using DocumentFormat.OpenXml.Spreadsheet;
+using System.IO;
+using System.Linq;
+using Acceleratio.XCellKit.ExcelExport.Helpers;
+using Chart = DocumentFormat.OpenXml.Drawing.Charts.Chart;
+
+namespace Acceleratio.XCellKit.ExcelExport
+{
+    public class Charts : ReportExport
+    {
+        public Charts(ReportExport export)
+            : base(export)
+        {
+
+        }
+        /// <summary>
+        /// Creates, designes and places chart into a sheet.
+        /// Calls overrideable methods for displaying the chart based on ChartType.
+        /// </summary>
+        public void InsertChartInSpreadsheet(Stream stream)
```

Comment:
Also, i would like a utility function somewhere to anchor that drawing to a particular cell. Sometimes i may want the graph to be at (A6,B2), sometimes (A10,B1)